src/magic_folder/uploader.py

```diff
@@ -297,7 +297,7 @@ def upload_local_snapshots(self):
                 # Unable to reach Tahoe storage nodes because of network
                 # errors or because the tahoe storage nodes are
                 # offline. Retry?
-                print(Failure())
```

Comment:
Dang, does this really fix the deprecation builder??!

(Anyway, the reason I put that there is because I lost some time on a silly syntax error that was eating...and `Exception` is of course way too broad for "tahoe did a thing wrong" anyway)

Reply:
It makes the builder green, but as I explain in https://github.com/LeastAuthority/magic-folder/pull/439#pullrequestreview-692421764, it just makes the builder useless, as it won't catch any errors.

Reply:
Yeah, we definitely need better error handling here, but this will at least log the failure, and also cause unit tests that trigger it to fail.